ivy/container/elementwise.py

```diff
@@ -6245,3 +6245,59 @@ def erf(
         return self.static_erf(
             self, key_chains, to_apply, prune_unapplied, map_sequences, out=out
         )
+
+    @staticmethod
+    def static_minimum(
+        x1: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        x2: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        *,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        return ContainerBase.multi_map_in_static_method(
+            "minimum",
+            x1,
+            x2,
+            key_chains=key_chains,
+            to_apply=to_apply,
+            prune_unapplied=prune_unapplied,
+            map_sequences=map_sequences,
+            out=out,
+        )
+
+    def minimum(
+        self,
+        x2,
+        key_chains=None,
+        to_apply=True,
+        prune_unapplied=False,
+        map_sequences=False,
+        out=None,
+    ):
+        """Computes the elementwise minimum between this container and another container
+        or number.
+
+        Parameters
+        ----------
```

Comment:
missing self parameter, you can refer to functional `minimum` for description :)